docs/README.md

```diff
@@ -202,6 +207,23 @@ prerelease or remove it to prerelease all of them._
 
 </details>
 
+<details>
+<summary>Using Github Actions to Publish a Pre-release</summary>
```

Comment:
I'm also wondering - if only we (team Atlantis) can pre-release using `npm run publish:prerelease`, is it just clutter to still have that information in the readme?

Should we only have the instructions to use GitHub actions and have all this information also stored in a Confluence page? I'm open either way, just wondering your thoughts...

Reply:
I think it is only Atlantis. I might mark it as that but I think unifying the docs makes sense. If someone on is looking how to do it I think it makes sense to have it in here (other libraries do it too) instead of having to open confluence and find the page

Reply:
Okay sounds good - marking it as such might just bring a little more clarity. Thanks Michael!